src/lib/Sympa/DataSource/LDAP.pm

```diff
@@ -108,7 +125,13 @@ sub _load_next {
     }
 
     my @retrieved;
-    my $mesg = $self->__dsh;
+    my $mesg;
+    if ($self->{_page} and $self->{_page}->cookie) {
```

Comment:
The cookie can be a string `"0"` by which this condition fails.  See [an issue on 389DS](https://pagure.io/389-ds-base/issue/47610).